specification/security/resource-manager/Microsoft.Security/preview/2022-01-01-preview/ServerVulnerabilityAssessmentsSettings.json

```diff
@@ -0,0 +1,328 @@
+          }
+        },
+        "tags": [
+          "ServerVulnerabilityAssessmentsSettings"
+        ],
+        "description": "Get a server vulnerability assessments setting of the requested kind, that is set on the subscription",
+        "operationId": "ServerVulnerabilityAssessmentsSettings_Get",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/SettingKind"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ServerVulnerabilityAssessmentsSetting"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
+        "x-ms-examples": {
+          "Set a server vulnerability assessments setting of the kind settingKind on the subscription": {
+            "$ref": "./examples/ServerVulnerabilityAssessmentsSettings/PutServerVulnerabilityAssessmentsSetting_example.json"
+          }
+        },
+        "tags": [
+          "ServerVulnerabilityAssessmentsSettings"
+        ],
+        "description": "Create a server vulnerability assessments setting of the requested kind on the subscription",
+        "operationId": "ServerVulnerabilityAssessmentsSettings_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/SettingKind"
+          },
+          {
+            "$ref": "#/parameters/SettingBody"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ServerVulnerabilityAssessmentsSetting"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Delete the server vulnerability assessments setting of the kind settingKind from the subscription": {
+            "$ref": "./examples/ServerVulnerabilityAssessmentsSettings/DeleteServerVulnerabilityAssessmentsSetting_example.json"
+          }
+        },
+        "tags": [
+          "ServerVulnerabilityAssessmentsSettings"
+        ],
```

Comment:
If you can delete them, you can probably create them right? And vice versa. So I'd expect either there's no DELETE. and no PUT, but just PATCH, or PUT can return 201.